src/Tag/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { useLayer } from "react-laag";
+import cc from "classcat";
+import AsElement from "../util/AsElement";
+
+/**
```

Comment:
✂️ If you update this big jsDoc comment and put it directly above the `const Tag` function, the docs will show up in storybook above the "overview" story

Reply:
Wrote a new description. Will defer to any suggestions you have!